relays/utils/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "relay-utils"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+
+[dependencies]
+async-std = "1.6.2"
+backoff = "0.2"
+futures = "0.3.5"
+log = "0.4.11"
+sysinfo = "0.15"
+
+# Substrate dependencies
+substrate-prometheus-endpoint = { version = "0.8.0-rc6", git = "https://github.com/paritytech/substrate.git", tag = "v2.0.0-rc6" }
```

Comment:
To avoid long lines, can you use the `[dependencies.substrate-prometheus-endpoint]` style for imports?

Reply:
They're hard to read. When it's just one-line, I could easily see all dependencies on one page. When every dependency takes up 6 lines, it's easy to follow what crate really depends on. I'd prefer to have one-liners everywhere. But here I'm just starting with relay libs + bin.

Reply:
> I'd prefer to have one-liners everywhere.

Me too actually. The list of deps (names) is imho way more important than the rest of the line, it has to be always the same anyway (same version / git repo / etc).

Reply:
Maybe it's just me then, but I really don't like how these long lines look on (for example) Polkadot's `Cargo.toml` files. It's especially unhelpful when the feature flags are declared at the end and you need to scroll around to see what features a particular crate has.

That being said, I think we'll get off the GitHub versions in a few weeks and start using versions directly off crates.io. At that point I wouldn't mind in-line dependencies.